.ebextensions/options.config

```diff
@@ -0,0 +1,3 @@
+option_settings:
+  aws:elasticbeanstalk:application:environment:
+    NODE_ENV: production
```

Comment:
I don't know what this actually does or why it's here, but it was in fusion-curation, soooo